src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+public class DukeException extends Exception {
+    /**
+     *
+     * @param message shows error message for DukeException
```

Comment:
Maybe separate the description of the method (storing the error message) from the purpose of the parameter (error message itself)? Since the exception does not do any showing of the error message.